src/corehost/cli/apphost/CMakeLists.txt

```diff
@@ -20,8 +20,13 @@ set(SKIP_VERSIONING 1)
 
 set(SOURCES
     ../fxr/fx_ver.cpp
+    ./bundle/bundle_runner.cpp
+    ./bundle/manifest.cpp
+    ./bundle/file_entry.cpp
 )
 
+include_directories(./bundle)
```

Comment:
Do we need this? Can't we use relative paths to include from the corehost (and everything else is in the same directory)?